### PR TITLE
Add Russian localization for README

### PR DESCRIPTION
**Pull request by qweme32**
# What does this PR do?

Add Russian localization for README.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests)

### CODE REVIEW

**Comment by ArthurZucker** [comment]
cc @stevhliu

**Comment by kawaiioi** [comment]
awesome

**Comment by qweme32** [comment]
any comments?

File: README_ru.md [hunk 1]
```diff
@@ -0,0 +1,548 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
```
Comment (stevhliu) [inline README_ru.md:2]:
```suggestion
Copyright 2023 The HuggingFace Team. All rights reserved.
```

File: README_ru.md [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+1. **[ViT Hybrid](https://huggingface.co/docs/transformers/model_doc/vit_hybrid)** (from Google AI) released with the paper [An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale](https://arxiv.org/abs/2010.11929) by Alexey Dosovitskiy, Lucas Beyer, Alexander Kolesnikov, Dirk Weissenborn, Xiaohua Zhai, Thomas Unterthiner, Mostafa Dehghani, Matthias Minderer, Georg Heigold, Sylvain Gelly, Jakob Uszkoreit, Neil Houlsby.
+1. **[VitDet](https://huggingface.co/docs/transformers/model_doc/vitdet)** (from Meta AI) released with the paper [Exploring Plain Vision Transformer Backbones for Object Detection](https://arxiv.org/abs/2203.16527) by Yanghao Li, Hanzi Mao, Ross Girshick, Kaiming He.
+1. **[ViTMAE](https://huggingface.co/docs/transformers/model_doc/vit_mae)** (from Meta AI) released with the paper [Masked Autoencoders Are Scalable Vision Learners](https://arxiv.org/abs/2111.06377) by Kaiming He, Xinlei Chen, Saining Xie, Yanghao Li, Piotr Dollár, Ross Girshick.
```
Comment (stevhliu) [inline README_ru.md:493]: Let's also add ViTMatte here

```suggestion
1. **[ViTMAE](https://huggingface.co/docs/transformers/model_doc/vit_mae)** (from Meta AI) released with the paper [Masked Autoencoders Are Scalable Vision Learners](https://arxiv.org/abs/2111.06377) by Kaiming He, Xinlei Chen, Saining Xie, Yanghao Li, Piotr Dollár, Ross Girshick.
1. **[ViTMatte](https://huggingface.co/docs/transformers/main/model_doc/vitmatte)** (from HUST-VL) rreleased with the paper [ViTMatte: Boosting Image Matting with Pretrained Plain Vision Transformers](https://arxiv.org/abs/2305.15272) by Jingfeng Yao, Xinggang Wang, Shusheng Yang, Baoyuan Wang.
```

**Review by stevhliu — COMMENTED** [review]
Thanks for your translation!

**Comment by qweme32** [comment]
thx, added @stevhliu

**Review by stevhliu — APPROVED** [review]
Thanks again! 🤗

**Pull request merged**